.github/actions/pull_images/action.yml

```diff
@@ -29,7 +33,7 @@ runs:
       shell: bash
       run: |
         source venv/bin/activate
-        python utils/scripts/get-image-list.py '${{ inputs.scenarios }}' > compose.yaml
+        python utils/scripts/get-image-list.py '${{ inputs.scenarios }}' ${{ inputs.exclude_buddies_images }} > compose.yaml
```

Comment:
This issue should have been fixed by #2704, let me know if you have trouble with this point.